### PR TITLE
docs: add biweekly report

### PR DESCRIPTION
**Pull request by Wenjie0329**

### CODE REVIEW

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# 2022.11.24 ~ 2022.12.9 —— Allowing S3 Storage
+
+December 15, 2022 · 6 min read
+
+>Since open-sourced on Nov.15, the GreptimeDB community has merged 21 PRs from 10 contributors. That's nearly 25% of the whole project. What a productive month! This article updates two of the good first issues that we are working on and two of the highlighted features.
+
+## Introduction
+
+It's only been three weeks and the community has already received lots of attention and powerful support from developers scattered around the world. Their talent and passion took us by surprise and strengthened our belief in open source.
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:9]:
```suggestion
It's only been three weeks and the community has already received lots of attentions and powerful support from developers around the world. Their talent and passion took us by surprise and strengthened our belief in open source.
```

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Contributors list: (In alphabetical order)
+
+[@azhsmesos](https://github.com/azhsmesos) [@DiamondMofeng](https://github.com/DiamondMofeng) [@dongxuwang](https://github.com/dongxuwang)
+
+[@e1ijah1](https://github.com/e1ijah1) [@francis-du](https://github.com/francis-du) [@lbt05](https://github.com/lbt05)
+
+[@lizhemingi](https://github.com/lizhemingi) [@morigs](https://github.com/morigs) [@SSebo](https://github.com/SSebo) [@Xuanwo](https://github.com/Xuanwo)
+
+A big THANK YOU for the generous and brilliant contributions! It is people like you who are making GreptimeDB a great product. Let's keep up the good work!
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:25]:
```suggestion
A big THANK YOU for the generous and brilliant contributions! It is people like you who are making GreptimeDB a great product. Let's keep up with the wonderful work!
```

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+For `TIMESTAMP` column, a simplification is to change the grammar to:
+
+```SQL
+ts TIMESTAMP TIME INDEX,
+```
+
+By doing so, it
+
+- need not to write `NOT NULL`, this is implied by `TIME INDEX` keywords
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:60]:
```suggestion
- does not need to write `NOT NULL`, this is implied by `TIME INDEX` keywords
```

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+- need not to specify `TIME INDEX` in another line
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:61]:
```suggestion
- does not need to specify `TIME INDEX` in another line
```

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+GreptimeDB now supports both of these two grammar rules. Users can choose one of them according to their preference.
+
+### Issue #602 (**Help Wanted**)
+
+Link:  https://github.com/GreptimeTeam/greptimedb/issues/602
+
+Issue description:
+
+Add system tables for inner metrics.
+It's good to have a metrics table like the one in [Databend](https://databend.rs/doc/sql-reference/system-tables/system-metrics) or [ClickHouse](https://clickhouse.com/docs/en/operations/system-tables/metrics/). Users can easily "select" on the metrics table to see DB's important running status. With the metrics table, we can integrate other visualization tools or dashboards that support SQL like Grafana, to export some meaningful time series data.
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:72]: With the metrics table, we can integrate other visualization tools or dashboards that support SQL like Grafana, to export some meaningful time series data.  
```please review this sentence above, feel like the sub is missing

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+1. In datanode, create a new table called `metrics`, with a schema including at least `metric`, `value`, and `description`, and also feel free to add other columns when necessary. Register it into the default catalog as our `ScriptsTable` does.
+2. Implement "select" for the metrics table. Instead of actually storing the metrics, call the `render()` method of `PROMETHEUS_HANDLE`, and parse the result (Prometheus lines) as table output.
+3. In frontend, create the same metrics table as well. Unlike `DistTable`, the metrics table does not do distributed queries.
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:75]: Unlike `DistTable`, the metrics table does not do distributed queries.
```suggest to change "do" distributed queries to other verbs, such as, does not execute

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+### New things
+
+- In the latest [release of gtctl](https://github.com/GreptimeTeam/gtctl/releases/tag/v0.1.0-alpha.7), users can customize mirror repository.
+- We are now developing a dashboard for GreptimeDB. It's still at a very early stage and has a basic UI. Feel free to have a go. Visit https://github.com/GreptimeTeam/dashboard
+
+## Recent Blog Recommendations
+
+[Have a Grep Time, Making the Most of 2023 - GreptimeDB Roadmap for Next Year | Greptime Blog](https://greptime.com/blogs/2022-11-29-have-a-grep-time-making-the-most-of-2023.html)
+
+This roadmap presents goals our team is working on and concepts we have for our shared community. It gives people a sense of what to expect in GreptimeDB over the next 6 months. Furthermore, it helps provide "starting points" for how to get involved, and a sense of what kind of projects we are looking for.
```
Comment (apdong2022) [inline docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:106]:
```suggestion
This roadmap presents goals our team is working on and concepts we have for our shared community. It gives people a sense of what to expect in GreptimeDB over the next 6 months. Furthermore, it provides a "starting points" for how to get involved, and a sense of what kind of projects we are looking for.
```

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md (hunk 5 above)
Comment (apdong2022) [reply docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:72]: sub is unclear

File: docs/changelog/biweekly-report/2022-12-16-biweekly-report.md (hunk 6 above)
Comment (xtang) [reply docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:75]: @apdong2022 it seems the suggestions are lost.

Comment (apdong2022) [reply docs/changelog/biweekly-report/2022-12-16-biweekly-report.md:75]: > @apdong2022 it seems the suggestions are lost.

please check again

**Review by xtang — APPROVED** [review]

**Review by apdong2022 — COMMENTED** [review]
lgtm

**Pull request merged**